packages/core/docs/recipes/authentication.md

```diff
@@ -1,3 +1,135 @@
 # Authentication
 
-Soon you will see here an authentication recipe.
+## Before start
+
+First of all, you must go through [Quick Start Guide](quick-start.md).
+
+## Configuring Authentication
+
+#### Configure strategy
+
+```sh
+└── src                              # application source code
+    └── auth                         # authentication strategies directory
+        ├── index.js                 # exports strategies
+        └── strategies
+            └── demoStrategy.js      # demo strategy
+```
+
+```js
+// demoStrategy.js
+const { HttpStatusError } = require('common-errors');
+
+function verifyToken(token) {
+  const validToken = !!token; // check token structure, check token on exist
+
+  if (!validToken) {
+    throw new HttpStatusError(403, 'Invalid Token');
+  }
+}
+
+function demoStrategy(request) {
+  const { action } = request;
+  const { auth } = action;
+  const { strategy = 'required' } = auth;
```

Comment:
Please update the recipe according to the changes you introduced into https://github.com/microfleet/create-microfleet-app/pull/7